x/epochs/abci_test.go

```diff
@@ -22,95 +22,78 @@ func TestEpochInfoChangesBeginEndBlockersAndInitGenesis(t *testing.T) {
 	tests := []struct {
 		expCurrentEpochStartTime time.Time
 		expCurrentEpoch          int64
-		expCurrentEpochEnded     bool
+		expEpochStartTime        time.Time
 		fn                       func()
 	}{
 		{
 			expCurrentEpochStartTime: now,
-			expCurrentEpoch:          0,
-			expCurrentEpochEnded:     true,
-			fn: func() {
-				epochInfo = app.EpochsKeeper.GetEpochInfo(ctx, "monthly")
-			},
-		},
-		{
-			expCurrentEpochStartTime: now.Add(time.Second),
 			expCurrentEpoch:          1,
-			expCurrentEpochEnded:     false,
+			expEpochStartTime:        now,
 			fn: func() {
 				ctx = ctx.WithBlockHeight(2).WithBlockTime(now.Add(time.Second))
 				epochs.BeginBlocker(ctx, app.EpochsKeeper)
 				epochInfo = app.EpochsKeeper.GetEpochInfo(ctx, "monthly")
 			},
 		},
 		{
-			expCurrentEpochStartTime: now.Add(time.Second),
+			expCurrentEpochStartTime: now,
 			expCurrentEpoch:          1,
-			expCurrentEpochEnded:     false,
+			expEpochStartTime:        now,
 			fn: func() {
 				ctx = ctx.WithBlockHeight(2).WithBlockTime(now.Add(time.Second))
 				epochs.BeginBlocker(ctx, app.EpochsKeeper)
-				epochs.EndBlocker(ctx, app.EpochsKeeper)
```

Comment:
Can we undo deleting the end blocker here, want to have the test ensure it doesn't change anything

Reply:
Ah wait, we deleted the EndBlocker. So we should just delete this test entirely.

Reply:
Ah the function name might be confusing cause it still includes test endblocker. Fixed!